edge-modules/azure-monitor/src/IothubConnectionManager.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Client;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.Devices.Edge.Azure.Monitor
+{
+
+    public class IothubConnectionManager
+    {
+        ModuleClient ModuleClient;
+        public IothubConnectionManager(ModuleClient moduleClient)
+        {
+            this.ModuleClient = moduleClient;
+        }
+
+        public async Task ConnectToIothub()
+        {
+            try
+            {
+                await this.ModuleClient.OpenAsync();
+
+                LoggerUtil.Writer.LogInformation("Successfully re-established connection to IoT Hub");
```

Comment:
Does it matter if it logs `re-established` connection the first time it connects?